Makefile

```diff
@@ -1,17 +1,19 @@
 # Origin
-version_branch = v0.32
+version_branch = marko/4288_linting
```

Comment:
I assume this is just for testing purposes?

Reply:
yes this cant be merged until a different PR in tendemrint is merged and a release is cut with the changes.